Explorer/Assets/Scripts/Global/Dynamic/Bootstraper.cs

```diff
@@ -174,6 +183,48 @@ public async UniTask LoadStartingRealmAsync(DynamicWorldContainer dynamicWorldCo
             OpenDefaultUI(dynamicWorldContainer.MvcManager, ct);
         }
 
+        private bool DetectAndConfigureLocalSceneDevelopment(RealmLaunchSettings launchSettings)
```

Comment:
I wonder if we can/need to do in a more generalized way and move it from `Bootstrapper` to a separate class. And there is already some logic in `CloudBuild` connected to CLI arguments. Can we consider unifing them?

Reply:
I've just been checking the `CloudBuild` editor script usage and in there we use `Environment.GetEnvironmentVariables()`.

I've logged in a build both `Environment.GetEnvironmentVariables()` and `Environment.GetCommandLineArgs()`, and the first one only logs a lot of local OS environment variables (like the PATH in WindowsOS) but none of the command line arguments; Meanwhile the other one specifically gives us the local path to the application + the deep link string, where we can get the `realm`, `position` and any other parameter we want.

I've put the `LocalSceneDevelopmentController` in `DynamicWorldContainer` since there are other global-level controllers instantiated in there like `RealmController`, `RealUserInitializationFlowController`, `ChatMessageBus`, `ReloadSceneController` and others.

Since the local scene development flow has to affect the launch settings flow so that the application connects to the local scene realm, the more direct place I found to do such detection and configuration was in the new Bootstrapper itself.

Also for forcing raw GLTF loading on local scene development scenarios, we may need to inject some "localSceneDevelopment" bool into the StaticContainer eventually (loaded in also in the Bootstrapper), something we'll see when I integrate these changes in the Raw GLTF Loading branch.

Reply:
Let me see if I can modify the `DynamicWorldParams` to send the relevant data through there and run the `LocalSceneDevelopmentController` initialization inside the `DynamicWorldContainer` itself. That may be a bit cleaner.

Reply:
Check this new commit when you have the time: https://github.com/decentraland/unity-explorer/pull/1392/commits/51951a049b37036449be23ff31739819a5bf1c94